src/ports/postgres/modules/convex/utils_regularization.py_in

```diff
@@ -217,6 +217,11 @@ def __utils_normalize_data_grouping(y_decenter=True, **kwargs):
     """
     group_col = kwargs.get('grouping_col')
     group_col_list = split_quoted_delimited_str(group_col)
+    # If more than one column was specified for grouping, prefix each column
+    # name with '{tbl_data}.' to avoid ambiguity.
+    select_grouping_cols = ', '.join(['{tbl_data}.{grp_col}'.format(
```

Comment:
Isn't there a function in utilities that was added for this? Maybe use that one for simplicity?